phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixEmbeddedDriver.java

```diff
@@ -33,6 +33,10 @@
 
 import javax.annotation.concurrent.Immutable;
 
+import org.apache.phoenix.jdbc.bootstrapz.HBaseRegistryBootstrap;
```

Comment:
can we have just org.apache.phoenix.jdbc.bootstrap. ?
We generally don't use plurals in package names